### ISSUE TITLE
Update for 2.80?

### ISSUE THREAD

**Issue by tin2tin** [Pullusb/SceneSettings#1]
Are you planning to update this add-on to 2.80?

It looks like to could be useful to detect the difference between two files in this bug report: https://developer.blender.org/T70239

**Comment by Pullusb** [comment]
Hi @tin2tin !
I will certainly update it at some point. But I dont think it will be of much help for the bug report you mentionned. It's only a scene API attribute that are stored and compared, not preferences. (it might be possible to do preferences comparison too though, by tweking things a bit).
+ It's not possible to compare settings from 2.79 and 2.80 as API has changed.

Off topic : Will you be at the Blender conference tomorow ? I'll be there ;)

**Comment by tin2tin** [comment]
No, I'm not going. Enjoy it(and spread the idea that the Text Editor could use some more love... ;-) )
